dask/dataframe/io/parquet/arrow.py

```diff
@@ -1102,13 +1133,13 @@ def _collect_dataset_info(
             "metadata_task_size": metadata_task_size,
             "kwargs": {
                 "dataset": _dataset_kwargs,
-                "convert_string": config.get("dataframe.convert_string"),
+                "convert_string": dask.config.get("dataframe.convert_string"),
                 **kwargs,
             },
         }
 
     @classmethod
-    def _create_dd_meta(cls, dataset_info, use_nullable_dtypes=False):
+    def _create_dd_meta(cls, dataset_info):
```

Comment:
This will probably break `dask-geopandas`, but I'd rather make a fix in there than preserve an unused positional arg for compatibility.

Reply:
https://github.com/geopandas/dask-geopandas/pull/242